pages/admin/lessons/new/index.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react'
+import { gql, useLazyQuery } from '@apollo/client'
+import { AdminLessonCard } from '../../../../components/admin/lessons/AdminLessonCard'
+import { withGetApp, GetAppProps } from '../../../../graphql'
+import { AdminLayout } from '../../../../components/admin/AdminLayout'
+import Link from 'next/link'
+import styles from '../../../../scss/adminLessonPage.module.scss'
+
+const EXERCISES = gql`
+  query {
+    exercises {
+      flaggedAt
+      module {
+        lesson {
+          title
+        }
+      }
+    }
+  }
+`
+type Exercise = {
```

Comment:
Looks like we already have a type Exercise
https://github.com/garageScript/c0d3-app/blob/4e30845fb7b47ceca0a3e3c2733a61fb723d4776/graphql/index.tsx#L75

Is is possible use this type instead?